### ISSUE TITLE
annotations URL

### ISSUE THREAD

**Issue by Ppantaleo** [asmecher/hypothesis#22]
Hi Alec, I see that the plugin Scielo Preprints uses has this section: https://preprints.scielo.org/index.php/scielo/annotations
But I install it on a server I am setting up, I load a comment on the PDF and I can't access this URL: https://domain/server-name/annotations

I have removed in index.php from my URL

Thanks

**Comment by JhonathanLepidus** [comment]
Hi!

The annotations page is a feature we have developed to be used in the SciELO Preprints server. However, we didn't send it to be available in the original plugin yet. Can you confirm if you're using the original Hypothesis plugin?

**Comment by Ppantaleo** [comment]
Hello, thank you for your reply. Yes, I am implementing it in a preprint soon to be launched and I would be interested in adding that feature as well.

**Comment by diegoabadan** [comment]
Thanks @Ppantaleo .

Ideally, we'd like to integrate the changes we've made into the official plugin, if Alec/PKP agree.

There are three features:
* Add submission page with annotations
* Adds a button with the number of annotations next to the PDF button. When this is clicked, the PDF is opened with the annotations tab opening automatically.
* Adds the same button to the preprint item in the list of preprints

@asmecher , would you like to receive a PR with these changes? Presently, we only have them for OPS 3.3.

**Comment by asmecher** [comment]
Yes, I'd be very happy to review a PR for this, thanks! It would make the plugin a lot more useful.

**Comment by Ppantaleo** [comment]
Great, thanks!
